sdk/containerregistry/azure-containerregistry/azure/containerregistry/_container_repository_client.py

```diff
@@ -114,35 +112,36 @@ def get_tag_properties(self, tag, **kwargs):
         :returns: :class:~azure.containerregistry.TagProperties
         :raises: :class:~azure.core.exceptions.ResourceNotFoundError
         """
-        # GET '/acr/v1/{name}/_tags/{reference}'
         return TagProperties._from_generated(  # pylint: disable=protected-access
             self._client.container_registry_repository.get_tag_properties(self.repository, tag, **kwargs)
         )
 
     def list_registry_artifacts(self, **kwargs):
         # type: (...) -> ItemPaged[RegistryArtifactProperties]
-        """List the registry artifacts for a repository
+        """List the artifacts for a repository
 
         :keyword last: Query parameter for the last item in the previous query
         :type last: str
-        :keyword n: Max number of items to be returned
-        :type n: int
+        :keyword page_size: Max number of items to be returned
+        :type page_size: int
         :keyword orderby: Order by query parameter
         :type orderby: :class:~azure.containerregistry.RegistryArtifactOrderBy
         :returns: ~azure.core.paging.ItemPaged[RegistryArtifactProperties]
         :raises: None
         """
-        raise NotImplementedError("Not implemented")
-        # TODO: turn this into an ItemPaged
-        # artifacts = self._client.manifests.get_list(
-        #     self.repository,
-        #     last=kwargs.get("last", None),
-        #     n=kwargs.get("n", None),
-        #     orderby=kwargs.get("orderby"),
-        # )  # ,
-        # # cls=lambda objs: [RegistryArtifacts._from_generated(x) for x in objs])
-
-        # return RegistryArtifactProperties._from_generated(artifacts)
+        # GET /acr/v1/{name}/_manifests
+        last = kwargs.pop("last", None)
+        n = kwargs.pop("page_size", None)
+        orderby = kwargs.pop("order_by", None)
+        return self._client.container_registry_repository.get_manifests(
+            self.repository,
+            last=last,
+            n=n,
+            orderby=orderby,
+            cls=lambda objs: [
+                RegistryArtifactProperties._from_generated(x) for x in objs  # pylint: disable=protected-access
+            ],
+        )
 
     def list_tags(self, **kwargs):
```

Comment:
`param`s are specified in the docstring, but it looks like they should be `keyword`?